CorsixTH/Lua/dialogs/edit_room.lua

```diff
@@ -1182,6 +1182,16 @@ function UIEditRoom:setDoorBlueprint(orig_x, orig_y, orig_wall)
     return x, y, x_mod, y_mod, wall
   end
   
+  local function BitAND(a,b)--Bitwise and
```

Comment:
Language extensions like this function belong in utility.lua

Please add spaces after commas, and between operators, matching the style of the surrounding code.